pyenv-win/bin/pyenv.bat

```diff
@@ -59,17 +55,16 @@ if exist "%exe%.bat" (
   exit /b
 )
 
-
-
 rem ====================================================================================
 :run
 
 rem update PATH to active version
-set "path=%bindir%;%bindir%\Scripts;%path%"
+if exist %bindir% set "path=%bindir%;%bindir%\Scripts;%path%"
 
 rem copy params to program precisely, preserving double-quotes and percents.
 rem this is the main fix for how 'exec' processing in pyenv.vbs did not
 rem correctly handle (get passed) params with percent chars in them
+set params=
```

Comment:
In Travis, we call `pyenv` to run `pytest` (via python shim). But in the unit tests, `params` still contains original args (`-m pytest etc.`).
This works, but this should not be needed with `endlocal`. 🤔